### ISSUE TITLE
crispresso and amplican needed

### ISSUE THREAD

**Issue by bgruening** [galaxyproject/tools-iuc#3915]
From a user.

1- crispresso:
https://github.com/pinellolab/CRISPResso2
https://crispresso.pinellolab.partners.org/submission

2-amplican
https://bioconductor.org/packages/release/bioc/html/amplican.html
https://github.com/valenlab/amplican

**Comment by ieguinoa** [comment]
I have a start for crispresso2 [here](https://github.com/ieguinoa/CRISPResso2_wrapper)

**Comment by bgruening** [comment]
@ieguinoa if @gallardoalba or the tools working group finds time, is it ok for you if we take this and push it to IUC?

**Comment by ieguinoa** [comment]
of course.  We actually use it in production so the options in the wrapper are tested. But the tool has  a lot of parameters so it does not have a 100% coverage, but ping me and can help with finishing it in IUC

**Comment by bgruening** [comment]
100% coverage is probably not needed but tests and such. Thanks @ieguinoa!

**Comment by gallardoalba** [comment]
I'll work on amplican.
